domain/tx/token.py

```diff
@@ -126,3 +122,84 @@ def from_dict(cls, dikt: dict) -> 'Token':
         :rtype: :py:class:`domain.tx.token.Token`
         """
         return from_dict(data_class=cls, data=dikt)
+
+
+@dataclass
+class TokenListItem:
+    """Item of a token list with token info
+
+    :param id: id hash of the token
+    :type id: str
+
+    :param name: token name
+    :type name: str
+
+    :param symbol: token symbol
+    :type symbol: str
+
+    :param timestamp: timestamp of token creation
+    :type timestamp: int
+
+    """
+    id: str
+    name: str
+    symbol: str
+    timestamp: int
+
+
+@dataclass
+class TokenList:
+    """List of tokens
+
+    :param total: total number of tokens
+    :type total: int
+
+    :param tokens: list of tokens
+    :type tokens: List[:py:class:`domain.tx.TokenListItem`]
+    """
+    total: int
+    tokens: List[TokenListItem]
+
+    def to_dict(self) -> dict:
+        """ Convert a TokenList instance into dict
+
+        :return: Dict representations of TokenList
+        :rtype: dict
+        """
+        return asdict(self)
```

Comment:
I feel like all of these `to_dict` and `from_dict` aren't really helping in any way. I'd be fine using `asdict` and `from_dict` directly, or at most making thin wrapper functions on a util module in case we want to customize some behavior. What do you think?